posthog/models/experiment.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Experiment(models.Model):
+    name: models.CharField = models.CharField(max_length=80)
+    description: models.CharField = models.CharField(max_length=400, null=True, blank=True)
+    team: models.ForeignKey = models.ForeignKey("Team", on_delete=models.CASCADE)
+
+    # Filters define the target metric of an Experiment
+    filters: models.JSONField = models.JSONField(default=dict)
+
+    # Parameters include configuration fields for the experiment: What the control & test variant are called,
+    # and any test significance calculation parameters
+    parameters: models.JSONField = models.JSONField(default=dict, null=True)
```

Comment:
The difference between parameters and filters is hazy.

What are all the valid parameters? Can we, should we tie these directly into the schema?

Reply:
I haven't yet figured out what all these would be, hence the JSONfield cop out. But I do know they don't need to be referenced outside the scope of an experiment, and we'd never query for experiments based on these parameters.

And depending on what things we want to show, they can increase by a lot (ex: confidence_intervals, variant names, kind of significance test to run(?) etc. etc. )

Reply:
That's fair - as long as we don't have any experiments in production (or even then) let's feel free to replace the dict with proper values though as they're easier to communicate forwards in time and migrate :)